hogql_parser/parser.cpp

```diff
@@ -237,6 +237,10 @@ class HogQLParseTreeConverter : public HogQLParserBaseVisitor {
     if (select_union_stmt_ctx) {
       return visit(select_union_stmt_ctx);
     }
+    auto tag_element_ctx = ctx->tagElement();
+    if (tag_element_ctx) {
+      return visit(tag_element_ctx);
+    }
     return visit(ctx->selectStmt());
```

Comment:
The order of evaluation here is different than in Python. There, `tagElement` is the last resort, here it takes precedence over `selectStmt`. I think it's better to stick to the former